keyboards/thevankeyboards/minivan/keymaps/josjoha/config.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+// place overrides here
+
+#define PERMISSIVE_HOLD
+
+#define RGBLIGHT_EFFECT_BREATHING
+
+
+/* Some defines meant to decrease the firmware size.
+ */
+//Disable old style macro handling: MACRO() & action_get_macro
+#define NO_ACTION_MACRO // This saves 320 bytes
+//disable calling of action_function() from the fn_actions array (deprecated)
+#define NO_ACTION_FUNCTION  // This saves 96 bytes
+    //#define LINK_TIME_OPTIMIZATION_ENABLE // Did not decrease firmware size when tested on 26 Jan 2020 
```

Comment:
>// Did not decrease firmware size when tested on 26 Jan 2020 

That's because this should be in the rules.mk, as `LTO_ENABLE = yes`, or `LINK_TIME_OPTIMIZATION_ENABLE = yes`.  This will enable LTO for the board, and will automatically define `NO_ACTION_MACRO` and `NO_ACTION_FUNCTION`. 

https://github.com/qmk/qmk_firmware/blob/5b91c3e0a0dc8152f69130cf047f3df0d0f94421/tmk_core/common.mk#L170-L179